cowprotocol/accounting/rewards/mainnet/unusual_slippage_query_2332678.sql

```diff
@@ -7,27 +7,43 @@
 --  {{significant_slippage_value}} -- the absolute threshold above which a tx is always flagged as high-slippage
 with
 results_per_tx as (
-    select * from "query_4070065(blockchain='ethereum',start_time='{{start_time}}',end_time='{{end_time}}',slippage_table_name='slippage_per_transaction')"
+    select * from "query_4070065(blockchain='{{blockchain}}',start_time='{{start_time}}',end_time='{{end_time}}',slippage_table_name='slippage_per_transaction')"
+),
+
+url_helper as (
+    select
+        case
+            when '{{blockchain}}' = 'ethereum' then 'eth'
+            else '{{blockchain}}'
+        end
 )
 
 select  --noqa: ST06
     rpt.block_time,
     concat(environment, '-', name) as solver_name,
     concat(
         '<a href="https://dune.com/queries/4059683',
-        '?blockchain=ethereum',
+        '?blockchain={{blockchain}}',
         '&start_time={{start_time}}',
         '&end_time={{end_time}}',
         '&slippage_table_name=raw_slippage_breakdown',
         '" target="_blank">link</a>'
     ) as token_breakdown,
-    rpt.tx_hash,
+    concat(
+        '<a href="https://phalcon.blocksec.com/explorer/tx/',
+        (select * from url_helper),
+        '/',
+        to_hex(rpt.tx_hash),
+        '" target="_blank">',
```

Comment:
Caught my eye that the `to_hex` makes us lose the `0x` prefix. Seems to work regardless but this is cleaner.

```suggestion
        '" target="_blank">0x',
```